### ISSUE TITLE
Module for image optimizations

### ISSUE THREAD

**Issue by kjhnns** [albrechtpeters/middleman-boilerplate#3]
I'd love to have a middleman module that creates search engine optimized thumbnails during the build process

**Comment by gregoralbrecht** [comment]
It seems like my first choice, imageoptim, [does not work with Middleman v4](https://github.com/plasticine/middleman-imageoptim/issues/46). So I guess we have to optimize images manually for the time being.

But there's built-in support for source sets, which is nice! [See the official documentation.](https://middlemanapp.com/advanced/file_size_optimization/#using-source-sets)

**Comment by kjhnns** [comment]
yeah i ran into the same issue, but fortunately there is a fixed fork 
 
https://github.com/hwhelchel/middleman-imageoptim
